numba/cuda/tests/cudapy/test_gufunc.py

```diff
@@ -183,6 +187,28 @@ def copy2d(A, B):
         copy2d(A, out=B)
         self.assertTrue(np.allclose(A, B))
 
+    def test_inefficient_guvectorize(self):
```

Comment:
I think this could do with a comment explaining that the test creates an inefficient use of the GPU because the inputs are all mapped onto a single thread in a single block.

Reply:
(Note: this is resolved, but I can't resolve the comment because I don't have write access to this repo)